packages/secret-rotator/src/helper.ts

```diff
@@ -0,0 +1,135 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import {iam_v1} from '@googleapis/iam';
+import {SecretManagerServiceClient} from '@google-cloud/secret-manager';
+import {logger} from 'gcf-utils';
+
+export class Helper {
+  iamClient: iam_v1.Iam;
+  secretManagerClient: SecretManagerServiceClient;
+  serviceAccountProjectId: string;
+  serviceAccountEmail: string;
+  secretManagerProjectId: string;
+  secretName: string;
+
+  constructor(
+    iamClient: iam_v1.Iam,
+    secretManagerClient: SecretManagerServiceClient,
+    serviceAccountProjectId: string,
+    serviceAccountEmail: string,
+    secretManagerProjectId: string,
+    secretName: string
+  ) {
+    this.iamClient = iamClient;
+    this.secretManagerClient = secretManagerClient;
+    this.serviceAccountProjectId = serviceAccountProjectId;
+    this.serviceAccountEmail = serviceAccountEmail;
+    this.secretManagerProjectId = secretManagerProjectId;
+    this.secretName = secretName;
```

Comment:
These instance variables look unused - each other method is using their method arguments.